### PR TITLE
added nervos blog post

### PR DESCRIPTION
**Pull request by Clearwood**
blog post detailing nervos announcement.

Closes https://github.com/leapdao/meta/issues/248

### CODE REVIEW

File: src/posts/nervos-announcement.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+description: Announcement, building layer-2 for Nervos CKB
+---
+
+We are excited to announce that we successfully applied for a [grant of the Nervos foundation](https://talk.nervos.org/t/ckb-sidechain-framework/4722) to work on second layer scaling for this blockchain.
+
+## A new kind of challenge
+
+The focus of Ethereum has been application development in the past, not necessarily scalability. The first turing-complete chain was a breakthrough but the high costs associated with operating on the main-net at the moment means that some ideas are not viable to be operated on it.
+
+We are and will be working on roll-up based solutions for Ethereum and will apply the same tactic towards Nervos. We find the transaction model interesting and want to apply our findings to a new ecosystem. We have been in contact with the part of the Nervos' team for years and are excited to work together with them.
```
Comment (mwaa) [inline src/posts/nervos-announcement.md:19]: `We are actively working` removes double `and` in sentence

Comment (Clearwood) [inline src/posts/nervos-announcement.md:19]:
```suggestion
We are actively working on roll-up based solutions for Ethereum and will apply the same tactic towards Nervos. We find the transaction model interesting and want to apply our findings to a new ecosystem. We have been in contact with the part of the Nervos' team for years and are excited to work together with them.
```

**Comment by troggy** [comment]
great structure 👍
Cannot comment for the content as I don't know enough e.g. I don't understand how UTXO model is good for "parallel processing" — in my experience it is the opposite when it comes to "shared" resources like smart-contracts. Where can I read more about that in Nervos?

File: src/posts/nervos-announcement.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  twitter: "@KenoBudde"
+  link: https://github.com/clearwood/
+description: Announcement, building layer-2 for Nervos CKB
+---
+
+We are excited to announce that we successfully applied for a [grant of the Nervos foundation](https://talk.nervos.org/t/ckb-sidechain-framework/4722) to work on second layer scaling for this blockchain.
+
+## A new kind of challenge
+
+The focus of Ethereum has been application development in the past, not necessarily scalability. The first turing-complete chain was a breakthrough but the high costs associated with operating on the main-net at the moment means that some ideas are not viable to be operated on it.
```
[the remark this answers is not in the document]
Comment (johannbarbie) [reply src/posts/nervos-announcement.md:17]: the paragraph seems unfinished. it is missing a conclusion, especially in relationship to the headline. what is the new challenge, if ethereum only caters to non-scalable dapps?

File: src/posts/nervos-announcement.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+We are actively working on roll-up based solutions for Ethereum and will apply the same tactic towards Nervos. We find the transaction model interesting and want to apply our findings to a new ecosystem. We have been in contact with the part of the Nervos' team for years and are excited to work together with them.
```
Comment (johannbarbie) [inline src/posts/nervos-announcement.md:19]: > our findings

our findings of what?

File: src/posts/nervos-announcement.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+Nervos has been created with its primary focus on scalability, it is a layer 1 that is intended to be used by layer 2 networks, similarly to the beacon chain of ETH 2.0.
+
+We became interested in contributing our knowledge of layer 2 scalability to a network that has been setup to accomodate layer 2 solutions from the get-go.
+
+Interesting for us are especially the [UTXO model employed by Nervos](https://xuejie.space/2020_03_20_what_do_we_mean_when_we_say_account_model/)
+
+- UTXO model puts less logic in the chain.
```
Comment (johannbarbie) [inline src/posts/nervos-announcement.md:27]:
```suggestion
- it puts less logic in the chain.
```

File: src/posts/nervos-announcement.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+- UTXO model allows parallel execution.
```
Comment (johannbarbie) [inline src/posts/nervos-announcement.md:28]:
```suggestion
- it allows parallel execution.
```

File: src/posts/nervos-announcement.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+- Generators can fill in the needed features to enable smart contracts.
+
+The Nervos Cell model is especially interesting, as it implements a state rent through inflation of CKBytes.
```
Comment (johannbarbie) [inline src/posts/nervos-announcement.md:31]: > Nervos Cell model

should be explained before used in the text

File: src/posts/nervos-announcement.md [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+
+We want to provide a simple entry point for applications to run on Nervos, therefore we will spend our time on developing a second layer for Nervos. As majority of todays smart contract developers are knowledgeable with Ethereum and its dev tools we have set out to create a sidechain where these skills can be applied and eventually existing decentralized applications can easily be migrated.
```
Comment (johannbarbie) [inline src/posts/nervos-announcement.md:33]: > developing a second layer for Nervos

colloquial, undifferentiated expression. Why not actually use wording from our grant application: https://talk.nervos.org/t/ckb-sidechain-framework/4722

Comment (johannbarbie) [inline src/posts/nervos-announcement.md:33]:
```suggestion
We want to provide a simple entry point for applications to run on Nervos, therefore we will spend our time on developing a second layer for Nervos. As majority of todays smart contract developers are knowledgeable with Ethereum and its dev tools. We have set out to create a sidechain where these skills can be applied and eventually existing decentralized applications can easily be migrated.
```

Comment (johannbarbie) [reply src/posts/nervos-announcement.md:33]: structure is this paragraph is confusing. maybe talk about smart contract developers first, then conclude a simple entry point for those developers to the nervos ecosystem from that.

File: src/posts/nervos-announcement.md [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+
+## Nervos design philosophy that we jive with
```
Comment (johannbarbie) [inline src/posts/nervos-announcement.md:35]: everything in this paragraph has already been mentioned in the previous one.

File: src/posts/nervos-announcement.md [hunk 9]
```diff
@@ -0,0 +1,52 @@
...
+We really nerd out about the following Nervos features.
+
+1. We love that the UTXO-model borrows itself to functional programming and is scalable through [parallel processing](https://medium.com/@sunflora98/utxo-vs-account-balance-model-5e6470f4e0cf).
+2. Nervos was built with layer-2 in mind, it lends flexibility to off-chain constructions.
+
+## What will the world get from this Grant?
+
+We believe that it should be easy to work with great technology. Therefore using this grant we will not only work on an EVM compatible side-chain but also provide tools Ethereum developers are already comfortable with. These include the following:
+
+1. A WebUI to define UDT or ERC20 token
```
Comment (johannbarbie) [inline src/posts/nervos-announcement.md:46]: > tools Ethereum developers are already comfortable with

UDT is specific to CKB, hence not known to Eth developers.

File: src/posts/nervos-announcement.md [hunk 10]
```diff
@@ -0,0 +1,52 @@
...
+2. A CLI tool for deployment to CKB or the sidechain
```
Comment (johannbarbie) [inline src/posts/nervos-announcement.md:47]: also CLI tools are specific to our product. The grant proposal specifically mentions that the CLI will be replaced by a UI in later stages, so not clear why it is pointed out here.

File: src/posts/nervos-announcement.md [hunk 11]
```diff
@@ -0,0 +1,52 @@
...
+3. SDK tools to interact with the sidechain.
+
+In doing so, releasing to our sidechain will seem very familiar to everyone already working with dApps. Starting from scratch our tools will give you an easy start and decrease time to market for dApps.
```
Comment (johannbarbie) [inline src/posts/nervos-announcement.md:50]: > releasing to our sidechain will

what does this mean?

Comment (johannbarbie) [reply src/posts/nervos-announcement.md:50]: > Starting from scratch our tools

gramar? meaning?

File: src/posts/nervos-announcement.md [hunk 12]
```diff
@@ -0,0 +1,52 @@
+---
+template: post.html
+title: "Got Scaling? Building layer-2 for Nervos CKB"
```
[the remark this answers is not in the document]
Comment (Clearwood) [reply src/posts/nervos-announcement.md:3]: The idea was for it to be a snowclone on Got Milk?

**Comment by Clearwood** [comment]
> i'm missing a couple of interesting aspects in this writeup:
> 
> * **4 stages of grant** we have chosen an agile approach, so that within 2-4 month first teams can start using the product. after that we convert the product to tightly integrate with nervos consensus, which will extend the security to the dapps created.
> * **token economics** different `token economy blueprints` will be deployable per mouseclick, and allow teams to launch products with short time to market. The [integration with the Nervos DAO](https://hackmd.io/mx52xPZuQyejWFItbmQmTg) is especially interesting .
> * **ethermint** a BFT based EVM chain that has been launched by chainsafe and operated in production by aragon is core of the architecture. can the modules we develop be used by other teams?

Thanks for the very detailed review :+1:

File: src/posts/nervos-announcement.md (hunk 12 above)
[the remark this answers is not in the document]
Comment (Clearwood) [reply src/posts/nervos-announcement.md:3]: I read `goto considered harmful` a couple of days ago and thought it would be fun to try to use a phrase myself.

**Comment by johannbarbie** [comment]
@Clearwood Nervos guys are  expecting this announcement since 2 weeks. do you need help? what can we do to speed it up?

**Comment by Clearwood** [comment]
> @Clearwood Nervos guys are expecting this announcement since 2 weeks. do you need help? what can we do to speed it up?

thank you for checking in on me,  I would appreciate pair programming if you are free. the revision has really taken me some time.

**Comment by johannbarbie** [comment]
@Clearwood what happened with @eshavkun ? he is no help? can you find some-one else? i wouldn't have time before the weekend, but this is quite urgent.

**Comment by Clearwood** [comment]
@johannbarbie Please do tell me if you feel like I underdelivered on this, I tried to work in your comments, gonna stop by tomorrow at the office.

**Comment by eshavkun** [comment]
@johannbarbie I see this discussion for the first time. The last thing I remember (and saw) regarding this is us finishing the the original text and submitting it. Checking th comments now

File: src/posts/nervos-announcement.md [hunk 13]
```diff
@@ -0,0 +1,57 @@
...
+date: 2020-07-09 17:00:00
+image: ckb.png
+author:
+  name: "@KenoBudde"
+  twitter: "@KenoBudde"
+  link: https://github.com/clearwood/
+description: Announcement, building layer-2 for Nervos CKB
+---
+
+We are proud to announce to have entered a [grantee relationship with the Nervos Foundation](https://talk.nervos.org/t/ckb-sidechain-framework/4722) to work on second layer scaling, a method to increase throughput by bundling transactions on a different network, for CKB. Transferring the advancements made in the Ethereum ecosystem will bring Nervos CKB to the next level of their scalability roadmap.
```
Comment (johannbarbie) [inline src/posts/nervos-announcement.md:13]:
```suggestion
We are proud to announce that we have entered a [grantee relationship with the Nervos Foundation](https://talk.nervos.org/t/ckb-sidechain-framework/4722). 

We will work on layer-2 scaling solution for the Nervos ecosystem. Utilizing the concept of rollups,  a method to increase throughput by bundling transactions we hope to bring Nervos CKB to the next level of its scalability roadmap.
```

File: src/posts/nervos-announcement.md (hunk 12 above)
[the remark this answers is not in the document]
Comment (johannbarbie) [reply src/posts/nervos-announcement.md:3]: a snowclone on this got milk? https://www.nytimes.com/2014/11/18/upshot/got-milk-might-not-be-doing-you-much-good.html

wouldn't the title need to be rather: `gas-storms considered harmful: building a layer-2 on top of Nervos CKB.`

[the remark this answers is not in the document]
Comment (Clearwood) [reply src/posts/nervos-announcement.md:3]: that is actually a pretty cool title, doing a snowclone on that is amazing <3

File: src/posts/nervos-announcement.md [hunk 14]
```diff
@@ -0,0 +1,59 @@
+---
+template: post.html
+title: "Got Scaling? Building layer-2 for Nervos CKB"
```
Comment (Clearwood) [inline src/posts/nervos-announcement.md:3]:
```suggestion
title: "gas-storms considered harmful: building a layer-2 on top of Nervos CKB"
```

File: src/posts/nervos-announcement.md [hunk 15]
```diff
@@ -0,0 +1,59 @@
+---
+template: post.html
+title: "gas-storms considered harmful: building a layer-2 on top of Nervos CKB"
```
Comment (zschavi) [inline src/posts/nervos-announcement.md:3]: Title leaves me confused...

We can keep it simple:
Announcement: LeapDAO building a layer2 for Nervos

**Comment by zschavi** [comment]
approved final draft, as requested changes by @johannbarbie  have been implemented.

* Title is misleading, should include both LeapDAO & Nervos to tease & gain attention through headline :)

File: src/posts/nervos-announcement.md [hunk 16]
```diff
@@ -0,0 +1,52 @@
...
+date: 2020-07-09 17:00:00
```
Comment (Clearwood) [inline src/posts/nervos-announcement.md:4]:
```suggestion
date: 2020-07-30 17:00:00
```

**Comment by Clearwood** [comment]
TL;DR; We are proud to announce that we have entered a grantee relationship with the Nervos Foundation. Over the next year, we will work on an EVM-compatible layer 2 solution for Nervos CKB based on Ethermint.

File: src/posts/nervos-announcement.md [hunk 17]
```diff
@@ -0,0 +1,53 @@
+---
+template: post.html
+title: "Gas-Storms Considered Harmful: Building a Layer-2 on top of Nervos CKB"
+date: 2020-07-09 17:00:00
```
Comment (Clearwood) [inline src/posts/nervos-announcement.md:4]:
```suggestion
date: 2020-08-06 17:00:00
```
